src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -208,6 +208,7 @@ class IpexCommunicationService extends AgentService {
     const msg = await this.props.signifyClient.exchanges().get(msgSaid);
     const schemaSaid = msg.exn.a.s;
     const attributes = msg.exn.a.a;
+    const recipient = msg.exn.a.i;
```

Comment:
`a.i` might be removed soon in KERI. Does `msg.exn.rp` exist with the same value? Should be something like that

(It will also get renamed to `ri` soon, so we have to remember to update to that, but not yet :P)